README.md

```diff
@@ -31,46 +31,47 @@
 
 # 📖 Nahid-s_Portfolio <a name="about-project"></a>
 
-Nahid-s_Portfolio is a HTML and CSS porject to develop personal portfolio wiith some mandatory features.
+This project is an online training and teaching website. Which will help people to get training and elegible persone to have a platform to showcase theire skills and expertise as a trainer.
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
 <details>
-  <summary>Software</summary>
+  <summary>Software used</summary>
   <ul>
     <li><a href=" ">HTML</a></li>
     <li><a href=" ">CSS</a></li>
+    <li><a href=" ">JavaScript</a></li>
   </ul>
 </details>
 
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-- Work Showcase
-- Personal Information
-- Contact information
+- Home Page
+- About page
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-To see the live demo of the page please click <a href="https://nahid1911.github.io/Nahid-s_Portfolio/">here</a>:
+<!-- To see the live demo of the page please click <a href="https://nahid1911.github.io/Nahid-s_Portfolio/">here</a>: -->
+The Page is under consturctin
```

Comment:
- [ ] [REQUIRED] The project deployed properly as expected but a `link` to the deployed project isn't present in the `README` please paste it 🙏🏼

Reply:
@mukhammadrizooff 
resolved

Reply:
@mukhammadrizooff 
Resolved